backend/tests/integration/orcid/cassettes/TestOrcidPusherPostNewWork.test_push_new_work_already_existing_and_delete_duplicated_records_different_putcodes.yaml

```diff
@@ -0,0 +1,411 @@
+      \    author = \"Rossoni, A.\",\n    doi = \"10.1111/test.orcid.push\",\n   \
+      \ month = \"5\",\n    title = \"{Some Results Arising from the Study of ORCID\
+      \ push}\",\n    year = \"2017\"\n}\n</work:citation-value></work:citation><work:type>journal-article</work:type><common:external-ids><common:external-id><common:external-id-type>other-id</common:external-id-type><common:external-id-value>45</common:external-id-value><common:external-id-url>http://inspirehep.net/record/45</common:external-id-url><common:external-id-relationship>self</common:external-id-relationship></common:external-id><common:external-id><common:external-id-type>doi</common:external-id-type><common:external-id-value>10.1111/test.orcid.push</common:external-id-value><common:external-id-url>http://dx.doi.org/10.1111/test.orcid.push</common:external-id-url><common:external-id-relationship>self</common:external-id-relationship></common:external-id></common:external-ids><work:url>http://inspirehep.net/record/45</work:url><work:contributors><work:contributor><work:credit-name>Rossoni,\
+      \ A.</work:credit-name><work:contributor-attributes><work:contributor-sequence>first</work:contributor-sequence><work:contributor-role>author</work:contributor-role></work:contributor-attributes></work:contributor></work:contributors></work:work>"
+    headers:
+      Accept:
+      - application/orcid+json
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '1584'
+      Content-Type:
+      - application/orcid+xml
+    method: POST
+    uri: https://api.orcid.org/v2.0/0000-0003-1134-6827/work
+  response:
+    body:
+      string: '{"response-code":409,"developer-message":"409 Conflict: You have already
+        added this activity (matched by external identifiers), please see element
+        with put-code 72275728. If you are trying to edit the item, please use PUT
+        instead of POST.","user-message":"There was an error when updating the record.
+        Please try again. If the error persists, please contact INSPIRE-HEP for assistance.","error-code":9021,"more-info":"https://members.orcid.org/api/resources/troubleshooting"}'
+    headers:
+      Access-Control-Allow-Origin:
+      - '*'
+      Cache-Control:
+      - no-cache, no-store, max-age=0, must-revalidate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '474'
+      Content-Type:
+      - application/orcid+json; qs=2;charset=UTF-8
+      Date:
+      - Fri, 17 Apr 2020 14:12:58 GMT
+      Expires:
+      - '0'
+      Pragma:
+      - no-cache
+      Server:
+      - nginx/1.16.1
+      X-Content-Type-Options:
+      - nosniff
+      X-Frame-Options:
+      - DENY
+      X-XSS-Protection:
+      - 1; mode=block
+    status:
+      code: 409
+      message: Conflict
```

Comment:
I had a second thought about this, but after thinking about it a bit more (and remembering tests are not isolated), it's fine :+1:

Reply:
Yes, the thing is that we are pushing the work which already exists (twice) in orcid, so to cache all author putcodes and delete duplicated record from orcid there must be 409 (WorkAlreadyExists) before.

Reply:
Yes, the thing is that we are pushing the work which already exists (twice) in orcid, so to cache all author putcodes and delete duplicated record from orcid there must be 409 (WorkAlreadyExists) before.